jenkins-ci/process.py

```diff
@@ -0,0 +1,69 @@
+import json
+import sys
+import os
+import argparse
+sys.path.append(os.path.join(os.path.dirname(os.path.realpath(__file__)), os.pardir))
+from lib import bisect_lib as cb
+# from lib import common_lib as cb
```

Comment:
move all the methods in bisect_lib   and make use of from lib import common_lib as cb  as in https://github.com/linuxci/linuxci/blob/master/lib/postprocess.py

Reply:
this was creating an error because comm_lib had to written in python3, that's why the methods are in bisect_lib